apps/src/componentLibrary/card/Card/index.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+import styles from './card.module.scss';
+
+interface CardProps {
+  children: React.ReactNode;
+  'data-testid': string;
+}
+
+export const Card = ({children, ...props}: CardProps) => {
```

Comment:
Way to go above and beyond and create this Card component! Good thing you took this story; I was going to hack it together with existing components and guess-and-check CSS 😂